src/test/software/amazon/qldb/QldbDriverImplTest.java

```diff
@@ -123,6 +124,16 @@ public void testBuildWitNullIonSystem() {
                                      .build());
     }
 
+    @Test
+    public void testBuildWithNullHttpClientBuilder() {
+        assertThrows(NullPointerException.class,
+                     () -> QldbDriver.builder()
+                                     .sessionClientBuilder(mockBuilder)
+                                     .ledger(LEDGER)
+                                     .httpClientBuilder(null)
+                                     .build());
```

Comment:
Not convinced of the utility of this test, keep it if you feel strongly about it.

Reply:
I don't feel strongly that we need this test, will remove it.